packages/api-explorer/src/Doc.jsx

```diff
@@ -394,7 +396,9 @@ Doc.propTypes = {
   oas: PropTypes.shape({}),
   oauth: PropTypes.bool.isRequired,
   onAuthChange: PropTypes.func.isRequired,
+  onDocRender: PropTypes.func.isRequired,
   onGroupChange: PropTypes.func.isRequired,
+  rendered: PropTypes.bool,
```

Comment:
Can you set a default for this on line 408?

Reply:
Sure